src/js/script-loader.js

```diff
@@ -721,17 +721,26 @@ var LoaderProtoMethods = {
                     dependencyImplementations.push(exportsImpl);
                 } else if (dependency === 'require') {
                     var localRequire = function(moduleName) {
-                        var mappedModuleName = configParser.mapModule(moduleName);
+                        var argc = arguments.length;
 
-                        for (var k = 0; k < module.dependencies.length; k++) {
-                            var dependency = module.dependencies[k];
+                        if (argc > 1) {
+                            var error = arguments[argc-1];
```

Comment:
No, they are not used. I will remove them and that will fix the previous issue too :+1: